starter-cli/src/main/java/io/micronaut/starter/cli/command/ListFeatures.java

```diff
@@ -94,6 +94,9 @@ private void listFeatures(ConsoleOutput consoleOutput, Set<Feature> defaultFeatu
             if (feature.isPreview()) {
                 name += " [PREVIEW]";
             }
+            if (feature.isCommunity()) {
```

Comment:
Can you add a test for this? A test which verifies the name is suffixed?

Reply:
Done!  PTAL https://github.com/micronaut-projects/micronaut-starter/pull/1063/commits/b189ca635115b2784f6b300e6a62b7f6f24064a7